packages/react-search-ui/src/containers/Sorting.js

```diff
@@ -44,7 +48,12 @@ export class SortingContainer extends Component {
       sortDirection,
       sortField,
       sortOptions,
-      view
+      view,
+      // eslint-disable-next-line no-unused-vars
+      results,
```

Comment:
Sorry if I'm wrong, but why we need to exclude `results` and `searchTerm`  from the `...rest` here?
It looks like for this file we could remove them both from `this.props` destructuring and from propTypes.
Does it make sense?